apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -91,65 +102,84 @@ const CustomizableCurriculumCatalogCard = ({
   quickViewButtonText,
   isEnglish,
   pathToCourse,
-}) => (
-  <div
-    className={classNames(
-      style.curriculumCatalogCardContainer,
-      isEnglish
-        ? style.curriculumCatalogCardContainer_english
-        : style.curriculumCatalogCardContainer_notEnglish
-    )}
-  >
-    <img src={imageSrc} alt={imageAltText} />
-    <div className={style.curriculumInfoContainer}>
-      <div className={style.labelsAndTranslatabilityContainer}>
-        <div className={style.labelsContainer}>
-          <CardLabels subjectsAndTopics={subjectsAndTopics} />
-        </div>
-        {isTranslated && (
-          <FontAwesome
-            icon="language"
-            className="fa-solid"
-            title={translationIconTitle}
-          />
-        )}
-      </div>
-      <h4>{courseDisplayName}</h4>
-      <div className={style.iconWithDescription}>
-        <FontAwesome icon="user" className="fa-solid" />
-        <p>{gradeRange}</p>
-      </div>
-      <div className={style.iconWithDescription}>
-        <FontAwesome icon="clock" className="fa-solid" />
-        <p>{duration}</p>
-      </div>
+  sectionsForDropdown,
+  ...props
+}) => {
+  const [isAssignDialogOpen, setIsAssignDialogOpen] = useState(false);
+
+  return (
+    <>
       <div
         className={classNames(
-          style.buttonsContainer,
+          style.curriculumCatalogCardContainer,
           isEnglish
-            ? style.buttonsContainer_english
-            : style.buttonsContainer_notEnglish
+            ? style.curriculumCatalogCardContainer_english
+            : style.curriculumCatalogCardContainer_notEnglish
```

Comment:
These are actually consistent with what was there earlier –– git is just confused